src/analyzer/protocol/dns/DNS.h

```diff
@@ -208,13 +213,38 @@ struct NSEC3_DATA {
 	VectorValPtr bitmaps;
 };
 
+struct NSEC3PARAM_DATA {
+	unsigned short nsec_flags;		// 8
```

Comment:
Indentation got sorta screwy here with this struct's comments.

Reply:
Fixed the indentation.